kernel/src/main/java/org/kframework/kast/KastModule.java

```diff
@@ -32,11 +35,16 @@ public void configure() {
         experimentalOptionsBinder.addBinding().toInstance(KastOptions.Experimental.class);
     }
 
-    @Provides
+    @Provides @RequestScoped
```

Comment:
why was this made request scoped?

Reply:
The @RequestScoped for `GlobalOptions` and other options objects which are just getters have no effect. I just thought it's good to mention it explicitly, as a for of documentation. Or mabye I was chasing a problem and thought this might be the solution.

For FileUtil, KPrint and TTYInfo it's good to avoid unnecessary instantiation. In case of FileUtil @RequestScoped does not work in KEq, because we want a different FileUtil for each definition, so it is overridden to `NO _SCOPE`.